Description: Inhibit XSane's click-through license
 Finally give up and inhibit the license click-through.
Author: Julien BLACHE <jblache@debian.org>
Forwarded: not-needed

Index: xsane-0.998/src/xsane.c
===================================================================
--- xsane-0.998.orig/src/xsane.c	2010-11-16 21:57:51.000000000 +0100
+++ xsane-0.998/src/xsane.c	2011-02-04 19:50:40.957016002 +0100
@@ -1909,11 +1909,11 @@
 /* ---------------------------------------------------------------------------------------------------------------------- */
 
 static int xsane_pref_restore(void)
-/* returns true if this is the first time this xsane version is called */
+/* Originally: returns true if this is the first time this xsane version is called */
 {
  char filename[PATH_MAX];
  int fd;
- int result = TRUE;
+ int result = FALSE;
  int i;
 
   DBG(DBG_proc, "xsane_pref_restore\n");
@@ -1927,6 +1927,7 @@
     preferences_restore(fd);
     close(fd);
 
+#if 0
     /* the version test only is done for the local xsane.rc file because each user */
     /* shall accept (or not) the license for xsane */
     if (preferences.xsane_version_str)
@@ -1936,6 +1937,7 @@
         result = FALSE; /* this version already has been started */
       }
     }
+#endif /* 0 */
   }
   else /* no local sane file, look for system file */
   {
@@ -5815,7 +5817,10 @@
     xsane.xsane_rc_set_filename = strdup("xsane"); /* ".rc" is appended later */
   }
 
+  xsane_pref_restore();
+  xsane_pref_save();
 
+#if 0 /* Original XSane code */
   if (xsane_pref_restore()) /* restore preferences, returns TRUE if license is not accpted yet */
   {
     if (xsane_display_eula(1)) /* show license and ask for accept/not accept */
@@ -5828,6 +5833,7 @@
       xsane_pref_save();
     }
   }
+#endif /* Original XSane code */
 
   xsane_pref_restore_media();
 
